misc/pootle-migration.md

```diff
@@ -0,0 +1,51 @@
+# FAQ: Shutting down Pootle and migrating to Pontoon
+
+## Q: What are the first things I need to do in order to adapt to Pontoon?
+
+There are a few things you can do to make the transition run smoothly:
+1. Log into Pontoon with your Firefox Account. If you don’t already have a Firefox account, please [create one](https://accounts.firefox.com/settings).
+2. Check your permissions on your profile to make sure they match what you’d expect.
+3. Flag issues with Pontoon to the l10n-drivers so that we can triage them and address them in a timely manner. To do this, please [file a bug here](https://goo.gl/tRQzAJ), or reach out to the l10n-drivers if you’re not yet comfortable with Bugzilla.
+
+If you’re a locale manager on Pootle, check the [community permissions](https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/users.html#managing-permissions) on Pontoon and make any necessary changes to mirror your community’s permission settings.
+
+## Q: When is my locale going to be migrated to Pontoon?
+
+We plan to migrate one project at a time to Pontoon, starting with those that don’t have upcoming deadlines. We’re going to announce the deadline for each project across our communication channels, in order to give you a chance to review pending suggestions.
+
+If your locale is up to speed, doesn’t have any pending suggestions, and you prefer to migrate all projects at once, please get in touch with l10n-drivers to coordinate the process.
+
+## Q: Will there be a training session, doc or video on how to use Pontoon?
+
+We’ve created this [set of documents](https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/) to instruct on the basics of using Pontoon. We’ll add more and more to these docs as time goes by. You’re also welcome to add to that doc by submitting pull requests to the [localizer documentation GitHub repository](https://github.com/mozilla-l10n/localizer-documentation).
+
+## Q: What will happen to my user profiles translation stats on Pootle?
+
+We have performed a database dump of all Pootle data. We’re working sort out how to migrate actions like submitted or approved translation attribution over to Pontoon.
```

Comment:
Should be "We’re working _to_ sort out how to migrate actions like submitted or approved translation attribution over to Pontoon."